Scripts/RemediationScripts/Remediate-SetClassicFrontDoorMinTLSVersion.ps1

```diff
@@ -0,0 +1,996 @@
+            $frontdoorEndpoint | Add-Member -NotePropertyName PreviousMinimumTlsVersion -NotePropertyValue $MinimumTlsVersion
+
+            Write-Host "Setting minimum required TLS Version for  Front Door : Resource ID: [$($resourceId)], Resource Group Name: [$($resourceGroupName)], Resource Name: [$($resourceName)]..." -ForegroundColor $([Constants]::MessageType.Info)
+            Write-Host $([Constants]::SingleDashLine)
+            
+            if (-not [System.Convert]::ToBoolean($isMinTlsVersionSetOnCustomDomain))
+            {
+                try
+                {
+                    Write-Host "Setting minimum required TLS version on custom domain :" $hostName -ForegroundColor $([Constants]::MessageType.Info)
+                    Write-Host $([Constants]::SingleDashLine)
+                    
+                    $resource = Enable-AzFrontDoorCustomDomainHttps -ResourceGroupName $resourceGroupName -FrontDoorName $resourceName -FrontendEndpointName $hostName -MinimumTlsVersion $requiredMinTLSVersion
+                    
+                    if($resource.MinimumTlsVersion -eq $requiredMinTLSVersion)
+                    {
+                        $frontdoorEndpoint.isMinTLSVersionSetOnCustomDomainPostRemediation = $true
+                        $frontdoorEndpoint.minimumTlsVersion = $resource.MinimumTlsVersion
+                        $customDomainsRemediated += $frontdoorEndpoint
+                        Write-Host "Minimum required TLS version for Front Door endpoint has been set successfully for :" $hostName -ForegroundColor $([Constants]::MessageType.Update)
+                        Write-Host $([Constants]::SingleDashLine)
+                    }
+                    else
+                    {
+                        $frontDoorsSkipped += $frontdoorEndpoint
+                        $logResource = @{}
+                        $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
+                        $logResource.Add("ResourceName",($_.ResourceName))
+                        $logResource.Add("Reason", "Error while setting the minimum required TLS version. Skipping this Front Door endpoint :" + $hostName )
+                        $logSkippedResources += $logResource
+                        Write-Host "Error while setting the minimum required TLS version for this Front Door Endpoint :" $hostName -ForegroundColor $([Constants]::MessageType.Error)
+                        Write-Host "Skipping this Front Door endpoint." -ForegroundColor $([Constants]::MessageType.Warning)
+                        Write-Host $([Constants]::SingleDashLine)
+                        return
+                    }
+                }
+                catch
+                {
+                    $frontDoorsSkipped += $frontdoorEndpoint
+                    $logResource = @{}
+                    $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
+                    $logResource.Add("ResourceName",($_.ResourceName))
```

Comment:
isMinTLSVersionSetOnCustomDomain

Reply:
It has been renamed to frontDoorsEpsSkipped